src/mongo/nodes.ts

```diff
@@ -211,11 +214,17 @@ export class MongoCollectionNode implements INode {
 		};
 	}
 
-	readonly command: Command = {
-		command: 'cosmosDB.openMongoCollection',
-		arguments: [this],
-		title: ''
-	};
+	readonly collapsibleState = vscode.TreeItemCollapsibleState.Collapsed;
+
+	async getChildren(): Promise<INode[]> {
+		const command = MongoCommands.getCommand(`db.${this.label}.find()`);
+		const parentNode = this;
+		let result = await MongoCommands.executeCommand(command, this.db).then(documents => JSON.parse(documents));
```

Comment:
The first three lines in this function can be simplified to:
```
let result = JSON.parse(await this.find());
```

Reply:
The functions find, insertOne, insertMany, etc. are all private. The way those methods and executeCommand are written seem to suggest that all the operations be done through the executeCommand method, which calls these private methods.

Reply:
As long as we're inside the class implementing the private functions, I think it's perfectly reasonable to use them